src/main/java/frc/robot/commands/Climb.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Climber;
+import frc.robot.Constants;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class Climb extends Command {
+  Climber globalClimber;
+
+  /** Creates a new Climb. */
+  public Climb(Climber passedClimber) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    globalClimber = passedClimber;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    globalClimber.setClimberMotorVelocity(Constants.consClimber.CLIMBER_MOTOR_VELOCITY);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    globalClimber.setClimberMotorVelocity(Constants.consClimber.CLIMBER_MOTOR_STOP);
```

Comment:
Since this is just 0 we do not have to make it a constant. So you can just put 0 and delete the constant